code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -173,13 +167,14 @@
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
 		if(!H.check_has_mouth())
+			flags -= NODROP
 			return
 
 	if(!sterile)
 		//target.contract_disease(new /datum/disease/alien_embryo(0)) //so infection chance is same as virus infection chance
 		target.visible_message("<span class='danger'>[src] falls limp after violating [target]'s face!</span>", \
 								"<span class='userdanger'>[src] falls limp after violating [target]'s face!</span>")
-
+		flags -= NODROP
```

Comment:
```suggestion
		flags &= ~NODROP
```
this is how we remove flags, it ensures that it doesnt try and remove a flag if it isnt there. (if flags = 0 and this line was ran, it would become negative)